posts/mpox-preparedness/index.qmd

```diff
@@ -0,0 +1,61 @@
+---
+title: "Epiverse-TRACE tools for analysing ongoing Mpox outbreak"
+author:
```

Comment:
Should probably be Epiverse-TRACE team.

Reply:
If we want this to be authored by the Epiverse-TRACE team, I imagine we need a different review process. Are you sure you want a collective author for this, and not have it be your blog post?

Reply:
Tagging @adamkucharski and @sbfnk here for thoughts.  I think this is more about providing an encompassing perspective and maybe a PI OR team article would be better than an individual's blogpost.

Reply:
Given this came out of a team discussion, think including (or inviting) all who contributed content would make sense, maybe with hard deadline for comments/review. 

And don't think job stage/RSE/academic split on authors matters much (here or elsewhere) – over the years, have seen great outbreak work by ECRs and developers...

Reply:
I'll incorporate current corrections and send out an open invite on Slack.